android/quest/src/test/java/org/smartregister/fhircore/quest/StructureMapUtilitiesTest.kt

```diff
@@ -455,4 +456,34 @@ class StructureMapUtilitiesTest : RobolectricTest() {
       }
     Assert.assertTrue(taskList.size == 10)
   }
+
+  @Test(expected = FHIRException::class)
```

Comment:
@DebbieArita I changed this to expect an exception, since I don't think the StructureMap is working yet, isn't that correct?